project-todo on request Mathilda

Code-lab-web opened the pull request:
Please include your Netlify link here.

Code-lab-web committed:
Code-lab-web

Code-lab-web committed:
Code-lab-web

Code-lab-web committed:
Add Redux DevTools extension, update Vite version, and implement Todo functionality

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Merge pull request #1 from Code-lab-web/draft/ecstatic-mendel

git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Merge branch 'main' of https://github.com/Code-lab-web/js-project-todo into draft/ecstatic-hermann

Code-lab-web committed:
Implement Todo form component and clean up unused state

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Merge pull request #2 from Code-lab-web/draft/ecstatic-hermann

git commit -m "Code-lab-web"

Code-lab-web committed:
Refactor App component, update todo store export, and fix TodoForm component formatting

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Merge branch 'main' of https://github.com/Code-lab-web/js-project-todo

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Add Card component with styling and hover effects

Code-lab-web committed:
Add dotenv dependency and create .env file for environment variables

Code-lab-web committed:
Add redirects configuration for improved URL handling

Code-lab-web committed:
Add express dependency to package.json

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Add CSS styles and configuration for MUI components and sliders

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Update MUI and styled-engine dependencies in package.json

Code-lab-web committed:
Add @fontsource/roboto dependency to package.json

Code-lab-web committed:
Add MUI icons material dependency and update font imports in CSS

Code-lab-web committed:
Update react-dom dependency to version 19.1.0

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Refactor CSS and add timeline component; clean up invalid code

Code-lab-web committed:
Add custom scrollbar theme and update CSS for color scheme support

Code-lab-web committed:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Code-lab-web committed:
Create README.md

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit-m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
Changes before Firebase Studio auto-run

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit-m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m ""

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit-m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "C"

Commit:
git coomit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

invalid-email-address committed:
Code-lab-web

invalid-email-address committed:
Clean up repository state

invalid-email-address committed:
git commit -m "Code-lab-web"

invalid-email-address committed:
git commit -m "Code-lab-web"

JennieDalgren reviewed (changes requested):
Read the requirements for this project and follow the guide on how to work with github! It is not possible to review this at all.

You are not approved on this project

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

HIPPIEKICK commented:
ping

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Commit:
git commit - m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Code-lab-web committed:
Create SECURITY.md for security policy

Added a security policy document outlining supported versions and vulnerability reporting.

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Add CRDA static analysis workflow

This workflow performs static analysis using Red Hat CodeReady Dependency Analytics, triggered on pushes, pull requests, and a schedule.

Commit:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"
